lib/usdUfe/utils/Utils.cpp

```diff
@@ -82,4 +82,20 @@ std::string prettifyName(const std::string& name)
     return prettyName;
 }
 
+std::vector<std::string> splitString(const std::string& str, const std::string& separators)
+{
+    std::vector<std::string> split;
+
+    std::string::size_type lastPos = str.find_first_not_of(separators, 0);
+    std::string::size_type pos = str.find_first_of(separators, lastPos);
+
+    while (pos != std::string::npos || lastPos != std::string::npos) {
+        split.push_back(str.substr(lastPos, pos - lastPos));
```

Comment:
I suppose this is moved code, but there is a bug if the string starts with separators. For example, assume the separator is ';' and the string is ";;;aaa;;;", then lastPos would be 3 (the first a) and pos would be zero (the first ;) and pos - lastPos would be 0 - 3, which when converted to unsigned would be a huge number.

Depending on the behavior of str.substr(), we either have abuffer overflow or we extract "aaa;;;" as the string.